test/Deployment.js

```diff
@@ -6,22 +8,22 @@ const config = {
         {
             symbol: 'XXX',
             decimals: 18,
-            supply: '1829101'
+            supply: 1_829_101
```

Comment:
come to think of it, what type is this in json?

Reply:
It needs to be a number (1829101) or a string ("1829101"), but it can be a `_` separated js literals :(

Reply:
weird, had no idea

Reply:
It hasn't...